docs/source/deployment/kubernetes/prerequisites.rst

```diff
@@ -0,0 +1,75 @@
+Corda Kubernetes Deployment option prerequisites
+================================================
+
```

Comment:
When I did a pull and made the HTML doc treee this page has weird formatting. I think this needs to be checked. 

<img width="1023" alt="Screen Shot 2020-01-24 at 11 24 00 AM" src="https://user-images.githubusercontent.com/38757983/73085127-34cea680-3e9c-11ea-8af1-4644a06408df.png">

Reply:
Do you mean the indented sections? The brief descriptions included in 3 of the options? Would it be better to indent the other 2 bulletpoints as well?